AUTHORS

```diff
@@ -6,3 +6,4 @@
 David Morgan/Google Inc. <davidmorgan@google.com>
 Matan Lurey/Google Inc. <matanl@google.com>
 Philipp Schiffmann <philippschiffmann93@gmail.com>
+Andrew Babin <code@andrewbab.in>
```

Comment:
Please sort alphabetically, and make sure there's a newline at the end of the file.  Thanks!